doc/Development_Documentation/01_Getting_Started/03_Configuration.md

```diff
@@ -2,9 +2,8 @@
 
 Pimcore's configuration can be found in several places:
 
-* Configurations in `var/config/*.(php|yml)` are written from the admin interface. For example the `system.yml` file contains
-  the settings from [System Settings](../18_Tools_and_Features/25_System_Settings.md)
-* The Symfony configuration tree (mainly distributed throughout `*.yml` files) contains all Symfony as well as most of the Pimcore related configurations.
+* Configurations in `var/config/*.(php|yml)` are written from the admin interface. For example the `system.yml` file contains the settings from [System Settings](../18_Tools_and_Features/25_System_Settings.md)
+* The Symfony configuration tree (mainly distributed throughout `*.yml` files) contains all Symfony as well as most of the Pimcore related configurations.)
```

Comment:
why the closing ")" ?

Reply:
You are right, the ")" is wrong here. I got confused because the live documentation is a bit off: 
https://pimcore.com/docs/5.x/Development_Documentation/Getting_Started/Configuration.html (The same bullet point has a starting "(" which needs to be closed).